README.md

```diff
@@ -1,3 +1,576 @@
+1. [iOS](#ios-1)
+    * [Add a public key to your app](#add-a-public-key-to-your-app)
+    * [Initialize the Framework](#initialize-the-framework)
+    * [Add custom URL scheme support to your application target](#add-custom-url-scheme-support-to-your-application-target)
+    * [Configure background tasks](#configure-background-tasks)
+1. [macOS](#macos-1)
+    * [Set an app bundle ID](#set-an-app-bundle-id) 
+    * [Add sandbox temporary exception entitlement](#add-sandbox-temporary-exception-entitlement)
+    * [Add public key to your app](#add-a-public-key-to-your-app-1)
+    * [Implement the release notes (What's New) functionality](#implement-the-release-notes-whats-new-functionality)
+    * [Add an email subscription form](#add-an-email-subscription-form)
+1. [Use the Vendor API to integrate apps into Setapp](#use-the-vendor-api-to-integrate-apps-into-setapp)
+1. [Logging](#logging)
+1. [Testing your app](#testing-your-app)
+
+# Integration requirements
+
+### iOS
+
+* The Setapp iOS Framework can be integrated into apps developed with iOS 10.0 or later. 
+The Framework doesn’t work with the watchOS and the tvOS yet.
+
+* The supported Swift version for the iOS Framework is 5.2 or later.
+
+* An iOS app must support custom URL schemes. The URL scheme must be the same as the app bundle ID.
+    Related documentation by Apple: [Defining a Custom URL Scheme for Your App](https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app).
+
+### macOS
+* The applications must be signed with a Developer ID certificate.
+* Compatibility with the latest macOS version must be tested and confirmed.
+
+    We don’t have strict requirements for supporting macOS versions. Setapp supports macOS versions from 10.15 (Catalina) to 12.0 (Monterey). However, we have a frozen version for our customers who use older than macOS 10.12 (Sierra) - 10.14 (Mojave) versions. So, if your app can support some of the older macOS versions - your revenue could increase.
+
+**Not allowed functionality of macOS apps:**
+* Paid features or app components;
+* Proprietary installer and update frameworks;
+* Activation and licensing mechanisms;
+* Built-in stores and in-app purchases.
+
+# Install and set up Framework
+
+## Installing the Framework
+
+### Swift Package Manager
+
+Linking the Framework with your project using the [Swift Package Manager](https://swift.org/package-manager/) requires Xcode 12 or later.
+
+Add the following dependency in your `Package.swift`:
+
+```swift
+dependencies: [
+  .package(name: "Setapp",
+           url: "https://github.com/MacPaw/Setapp-framework.git",
+           .branch("catalyst"))
+]
+```
+
+### CocoaPods
+
+With [CocoaPods](https://cocoapods.org/), add the following string to your `Podfile`:
+
+```ruby
+pod 'Setapp', :git => 'https://github.com/MacPaw/Setapp-framework.git', :branch => 'catalyst'
+```
+
```

Comment:
One question, do we need here branch `catalyst` if it will be merged to master? 

```suggestion
```ruby
pod 'Setapp', :git => 'https://github.com/MacPaw/Setapp-framework.git', :branch => 'catalyst'
```
```suggestion
```ruby
pod 'Setapp', :git => 'https://github.com/MacPaw/Setapp-framework.git', :branch => 'catalyst'
```

```
```

Reply:
Good one. Fixed.